ephemeral-hdfs/init.sh

```diff
@@ -17,22 +17,22 @@ case "$HADOOP_MAJOR_VERSION" in
     sed -i 's/-jvm server/-server/g' /root/ephemeral-hdfs/bin/hadoop
     ;;
   2) 
-    wget http://s3.amazonaws.com/spark-related-packages/hadoop-2.0.0-cdh4.2.0.tar.gz  
```

Comment:
I think our scripts are not fully compatible with yarn setup. Maybe is better to leave yarn) with hadoop-2.4